### PR TITLE
Add release GitHub actions workflow

### PR DESCRIPTION
**Pull request by tedpearson**
I'd like to use this! I'd love to have binaries available to easily download from GitHub rather than compiling locally. So I forked your project and threw together a release workflow.

- Updated build workflow versions
- Added (Manual) release workflow that builds for several os/arch targets
- Fixed a linting error about ioutil deprecation

Thanks for this project :)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+      - name: Create or update release
+        id: create_release
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          artifacts: ${{env.executable}}
+          tag: ${{inputs.release_version}}
+          commit: main
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (irvinlim) [inline .github/workflows/release.yml:50]: Any idea if I need to set this up manually in the repo settings?

**Review by irvinlim — APPROVED** [review]
Awesome, thanks for working on this and I'm glad you like the project! I'm currently away in vacation, will merge it once I'm back next week.

File: .github/workflows/release.yml (hunk 1 above)
Comment (tedpearson) [reply .github/workflows/release.yml:50]: There should be no need to set up anything, this is automatic: https://docs.github.com/en/actions/security-guides/automatic-token-authentication

**Comment by irvinlim** [comment]
@tedpearson I've merged the changes and done a release at v0.1.0, you can download the binaries from the GitHub releases page now!

**Comment by tedpearson** [comment]
Thanks @irvinlim - I installed this version and will close my fork for now :grinning:

**Pull request merged**